### ISSUE TITLE
"Clear" button in private mode is confusing

### ISSUE THREAD

**Issue by feiss** [MozillaReality/FirefoxReality#3702]
**Is your feature request related to a problem? Please describe.**
Private browsing mode top button says "CLEAR". That's confusing.. clear what?

**Describe alternatives you've considered**
- Better wording
- This button is quite odd, feels alien there. User can enter/exit private browsing with the tray button already, there shouldn't be the need for an additional button.

**Additional context**
![image](https://user-images.githubusercontent.com/359872/87820220-fc69bd00-c86d-11ea-8e0d-91c64aba3023.png)

**Comment by MortimerGoro** [comment]
I agree the button design is confusing. It offers a different feature to  private tab button:
- Private tab button: switches to regular tabs, but the private ones are kept open
- Clear button: Closes all private tabs

**Comment by feiss** [comment]
ohh, didn't realize that.. Confusing, yup. Tabs are confusing for me, in general. Unless you open the tab panel you are not aware of what is opened..
